meta/Meta.cpp

```diff
@@ -6702,6 +6702,7 @@ void Meta::meta_sai_on_port_state_change_single(
         case SAI_OBJECT_TYPE_PORT:
         case SAI_OBJECT_TYPE_BRIDGE_PORT:
         case SAI_OBJECT_TYPE_LAG:
+	case SAI_OBJECT_TYPE_TUNNEL:
```

Comment:
this code should be based on sai metadata, and was written when there was no metadata for those objecs yet, if you want to fix it, please add it using metadata checks

Reply:
i fixed it here: https://github.com/sonic-net/sonic-sairedis/pull/1469